allennlp_models/rc/models/transformer_qa.py

```diff
@@ -178,10 +185,19 @@ def forward(  # type: ignore
         ) + torch.gather(span_end_logits, 1, best_spans[:, 1].unsqueeze(1))
         best_span_scores = best_span_scores.squeeze(1)
 
+        best_span_probs = (
+            torch.gather(span_start_probs, 1, best_spans[:, 0].unsqueeze(1))
+            + torch.gather(span_end_probs, 1, best_spans[:, 1].unsqueeze(1))
+        ) / 2
```

Comment:
I don't think this is correct for probabilities. The probability of a span is the probability that it starts at the start token _times_ the probability that it ends at the end token.

Reply:
Good catch! I was going off of what the person said in the issue, but that makes more sense to me.